aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -200,6 +201,52 @@ Tensor rsub(const Tensor& self, Scalar other, Scalar alpha) {
   return native::rsub(self, wrapped_scalar_tensor(other), alpha);
 }
 
+Tensor& bitwise_xor_out(Tensor& result, const Tensor& self, const Tensor& other) {
+  auto iter = TensorIterator::binary_op(result, self, other,
+    /*check_mem_overlap=*/true);
+  bitwise_xor_stub(iter.device_type(), iter);
+  return result;
+}
+
+Tensor bitwise_xor(const Tensor& self, const Tensor& other) {
+  Tensor result = at::empty({0}, self.options());
+  native::bitwise_xor_out(result, self, other);
```

Comment:
Breaks XLA, please use `at::native::`
@ailzhang can we have use `at::native::` manifesto in wiki? ;)

Reply:
I believe this should be `at::` instead of `at::native::` :)

I've updated the PR